LICENSE

```diff
@@ -7,10 +7,10 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor: Babylonchain, Inc.
+Licensor: Babylon Labs, Ltd.
 
 Licensed Work: btc-staking-ts
-The Licensed Work is (c) 2023 Babylonchain, Inc.
+The Licensed Work is (c) 2024 Babylon Labs, Ltd.
```

Comment:
Need to verify whether this can be updated to 2024.

Reply:
Also, lines should be "-----------------------------------------------------------------------------" @vitsalis ?

Reply:
@vitsalis should I exclude license changes from the current PR, merge it, then create a separate PR once we get the info about the License?

Same for the `countdown`?

Reply:
Done